Contributing.md

```diff
@@ -25,6 +25,22 @@ The naming should be descriptive and direct, giving a clear idea of the function
 - .Net 8 SDK or newer
 - Visual Studio 2022 / JetBrains Rider 2023.3 or newer
 
+## Install addin from source
+
+- Clone this repository
+  - Or, fork and clone your fork if you plan to contribute
+- Ensure your development environment meets the
+  [prerequisites](#prerequisites-for-compiling-revitlookup) from the
+  contributing guidelines
+- Run `dotnet build -c '{configuration}'`
+  - Where `{configuration}` is a string defined in [`RevitLookup.csproj`][rl-proj] at the XPath `/Project/PropertyGroup/Configurations`
+  - E.g. `dotnet build -c 'Release R23'`
```

Comment:
with dotnet build SOLUTION configuration must be specified, not project configuration

`/Project/PropertyGroup/Configurations` not informative, because it does not indicate a valid path

Reply:
you can you instructions from here
https://github.com/Nice3point/RevitTemplates/blob/develop/samples/MultiProjectSolution/Readme.md

Reply:
> with dotnet build SOLUTION configuration must be specified, not project configuration

Hm, I'm not sure I understand the difference. I thought the configuration that the project received was inherited from the solution?

Though, maybe this would be more precise if it were:
```console
dotnet build source/RevitLookup/RevitLookup.csproj -c 'Release R23'
```

For example, in the repo root when I run:
```console
dotnet build -c 'Release R23'
```

I see the (trimmed) output:
<details>

```console
  Determining projects to restore...
  Restored ...\RevitLookup\source\RevitLookup\RevitLookup.csproj (in 1.27 sec).
  1 of 2 projects are up-to-date for restore.
  RevitLookup.UI -> ...\RevitLookup\source\RevitLookup.UI\bin\Release\net48\RevitLookup.UI.dll
  RevitLookup.UI -> ...\RevitLookup\source\RevitLookup.UI\bin\Release\net8.0-windows\RevitLookup.UI.dll
  RevitLookup -> ...\RevitLookup\source\RevitLookup\bin\Release R23\RevitLookup.dll
  RevitLookup -> ...\RevitLookup\source\RevitLookup\bin\Release R23\publish
  Publishing add-in RevitLookup -> ...\AppData\Roaming\Autodesk\Revit\Addins\2023\RevitLookup

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:12.44
```

</details>

...which indicates to me that `RevitLookup` has received the intended configuration, correct?

And dumping the MSBuild solution during `build` shows:
```xml
<ProjectConfiguration Project="{05C77115-2277-4DFC-8F95-BE37E5F1130F}" AbsolutePath="...\RevitLookup\source\RevitLookup\RevitLookup.csproj" BuildProjectInSolution="True">Release R23|AnyCPU</ProjectConfiguration>
```

---

> `/Project/PropertyGroup/Configurations` not informative, because it does not indicate a valid path

I believe this is a valid [XPath](https://developer.mozilla.org/en-US/docs/Web/XPath) within the XML of the project file. For example, when I run:

```console
xq -x /Project/PropertyGroup/Configurations .\source\RevitLookup\RevitLookup.csproj
```

I see:

```console
Debug R21;Debug R22;Debug R23;Debug R24;Debug R25
$(Configurations);Release R21;Release R22;Release R23;Release R24;Release R25
```

Reply:
> I thought the configuration that the project received was inherited from the solution?

no, https://github.com/jeremytammik/RevitLookup/blob/dev/RevitLookup.sln#L24-L39 solution has own configuration

Reply:
> For example, in the repo root when I run:> dotnet build -c 'Release R23'

Here `Release R23` must be solution configuration, not project

Reply:
> Though, maybe this would be more precise if it were: dotnet build source/RevitLookup/RevitLookup.csproj -c 'Release R23'

This line correct, Release R23 - project configuration, because you use path to csproj

Reply:
Ok, thank you! I will update.